POEApi.Model/Gear.cs

```diff
@@ -11,6 +11,9 @@ public class Gear : Item
         public List<Socket> Sockets { get; set; }
         public List<SocketableItem> SocketedItems { get; set; }
         public List<string> Implicitmods { get; set; }
+        public List<string> Craftedmods { get; set; }
```

Comment:
The `Item` base class already has properties for crafted mods and enchanted mods, so we shouldn't add them again here.  I'm thinking fractured mods should also be put in the `Item` class, but I'm not completely sure on that.

Reply:
I think fixed now